### PR TITLE
Added Tegro exchange

### PR DESCRIPTION
**Pull request by ashtegro**

### CODE REVIEW

**Comment by llamatester** [comment]
The tegro adapter exports: 
        
 
 _______________________________________
    Dexs for 25/9/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 50213.626215
    
    
    POLYGON 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0.09901
    Total volume: 22559.693464
    
    
    AVAX 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 1.24394
    
    
    BSC 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 1690126439739.783
    
    
    ARBITRUM 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 6.328742
    
    
    
    
    Done in 3.85s.

File: dexs/tegro/index.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+const getNumberOfDay = (date: Date) => {
+  const now: Date = new Date()
+  const timeDifference = now.getTime() - date.getTime()
+  return Math.floor(timeDifference / (24 * 60 * 60 * 1000))
+}
+
+const getVolume = async (queryUrl: string) => {
+  const start = new Date('1970-01-01T00:00:00Z')
```
Comment (dtmkeng) [inline dexs/tegro/index.ts]: daily volume is`id` dayid should pass from `timestamp`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The tegro adapter exports: 
        
 
 _______________________________________
    Dexs for 25/9/2023
    _______________________________________
    
    ETHEREUM 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 49955.04308
    Total volume: 50213.626215
    
    
    POLYGON 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 22559.693464
    
    
    AVAX 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 1.24394
    
    
    BSC 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 1690126439739.783
    
    
    ARBITRUM 👇
    Backfill start time not defined
    NO METHODOLOGY SPECIFIED
    Timestamp: 1695686398
    Daily volume: 0
    Total volume: 6.328742
    
    
    
    
    Done in 5.09s.

File: dexs/tegro/index.ts (hunk 1 above)
Comment (ashtegro) [reply dexs/tegro/index.ts]: Apologies, it is fixed now. Could you please check?

**Comment by dtmkeng** [comment]
@ashtegro  is it this protocol https://defillama.com/protocol/tegro-finance ?

**Comment by ashtegro** [comment]
@dtmkeng  No, it is not. I am also talking with Bentura on the DeFiLlama discord about this.  

Website: https://tegro.com
Twitter: https://twitter.com/tegrofi

**Pull request merged**